docs/proposals/20190919-machinepool-api.md

```diff
@@ -176,9 +176,6 @@ type MachinePoolSpec struct
     - Type: `MachineTemplateSpec`
     - Description: Machine Template that describes the configuration of each machine instance in a
       machine pool.
-  - **Strategy [optional]**
```

Comment:
This field does not exist. Given it wasn't implemented in the last 3-4 years it probably makes sense to remove it from the proposal

Reply:
Ah sorry, might be that we implement it now with MachinePool Machines. I'll update accordingly once I confirmed :) (still catching up..)

Reply:
Okay caught up. Sounds more like a future goal. So I think it's fine to drop it here to keep the proposal up-to-date. We can always re-add if we concretely plan do add this field.